python/cuspatial/cuspatial/core/interpolate.py

```diff
@@ -46,16 +46,16 @@ class CubicSpline:
     host based interpolation performance is likely to exceed GPU performance
     for a single curve.
 
-    cuspatial massively outperforms scipy however when many
-    splines are fit simultaneously. Data must be arranged in a SoA format,
-    and the inclusive/exclusive prefix_sum of the separate curves must also
-    be passed to the function.
+    However, cuSpatial massively outperforms scipy when many splines are fit
+    simultaneously. Data must be arranged in a SoA format, and the exclusive
+    `prefix_sum` of the separate curves must also be passed to the function.::
 
         t = cudf.Series(np.repeat(cp.arange(100), 1000)).astype('float32')
         y = cudf.Series(np.random.random(100*1000)).astype('float32')
         prefix_sum = cudf.Series(cp.arange(1000)*100).astype('int32')
-        new_samples = cudf.Series(np.repeat(np.linspace(0, 100, 1000), 1000)
-            .astype('float32'))
+        new_samples = cudf.Series(
+            np.repeat(np.linspace(0, 100, 1000), 1000)
+        ).astype('float32')
 
         curve = cuspatial.CubicSpline(t, y, prefixes=prefix_sum)
         new_points = curve(new_samples, prefix_sum*10)
```

Comment:
I tried running this example in a notebook, but I'm getting CUDA illegal address errors. Does this need updating for any recent API or implementation changes?

Reply:
@thomcom